src/app/CASESessionManager.cpp

```diff
@@ -146,7 +153,16 @@ void CASESessionManager::ReleaseAllSessions()
 CHIP_ERROR CASESessionManager::GetPeerAddress(const ScopedNodeId & peerId, Transport::PeerAddress & addr)
 {
     ReturnErrorOnFailure(mConfig.sessionInitParams.Validate());
-    auto optionalSessionHandle = FindExistingSession(peerId);
+    auto optionalSessionHandle = FindExistingSession(peerId, TransportPayloadCapability::kDefault);
+    ReturnErrorCodeIf(!optionalSessionHandle.HasValue(), CHIP_ERROR_NOT_CONNECTED);
+    addr = optionalSessionHandle.Value()->AsSecureSession()->GetPeerAddress();
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR CASESessionManager::GetPeerAddressForLargePayloadSession(const ScopedNodeId & peerId, Transport::PeerAddress & addr)
```

Comment:
nit: GetPeerAddressFromLargePayloadSession ?

Reply:
I think GetPeerAddressForLargePayloadSession() seems fine. I wanted it to mean to get the peer address for a session that supports large payload for the given PeerId.